Sources/NIOConcurrencyHelpers/NIOLockedValueBox.swift

```diff
@@ -45,9 +45,14 @@ public struct NIOLockedValueBox<Value> {
 
     /// Access the `Value`, allowing mutation of it.
     @inlinable
-    public func withNIOLockedValueBox<T>(_ mutate: (inout Value) throws -> T) rethrows -> T {
+    public func withLockedValue<T>(_ mutate: (inout Value) throws -> T) rethrows -> T {
         return try self._storage._lock.withLock {
             return try mutate(&self._storage._value)
         }
     }
 }
+
+#if compiler(>=5.5) && canImport(_Concurrency)
+extension NIOLockedValueBox: Sendable where Value: Sendable {}
+extension NIOLockedValueBox._Storage: @unchecked Sendable {}
```

Comment:
Sorry for nitpicking but is there any benefit making `_Storage` `@unchcked Sendable` instead of moving the `@unchecked` to the conditional conformance above and leaving `_Storage` `non-Sendable`?
I'm fine with either way as it is internal. Just merge it as we can revisit this at any time but I'm just curious

Reply:
thought I should use of the automatic inference where I can. Just in the (_very unlikely_...) case that somebody accidentally adds something else to `NIOLockedValueBox`, the compiler would now barf :P. It won't happen but you know.